app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -331,7 +333,7 @@ Vue.component('grantsCartEthereumPolygon', {
 
       let networkId = appCart.$refs.cart.networkId;
 
-      if (networkId !== '80001' && networkId !== '137' && appCart.$refs.cart.standardCheckoutInitiated == true) {
+      if (networkId !== '80001' && networkId !== '137' && appCart.$refs.cart.isCheckoutOngoing == true) {
```

Comment:
Can you replace the '80001' and '137' with variables or constants to avoid the [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))